src/components/helpdesk/repo/problem.repo.integration.test.js

```diff
@@ -0,0 +1,90 @@
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const { ObjectId } = require('mongoose').Types;
+const appPromise = require('../../../app');
+const testObjects = require('../../../../test/services/helpers/testObjects')(appPromise);
+const { problemRepo } = require('.');
+const { GeneralError, ValidationError } = require('../../../errors');
+
+chai.use(chaiAsPromised);
+const { expect } = chai;
+
+describe('problem repo', () => {
+	let app;
+	let server;
+
+	before(async () => {
+		app = await appPromise;
+		server = await app.listen(0);
+	});
+
+	afterEach(async () => {
+		await testObjects.cleanup();
+	});
+
+	after(async () => {
+		await server.close();
+	});
+
+	describe('deleteProblems', () => {
+		it('when problem is deleted, it should return deleted problem ids', async () => {
+			const user = await testObjects.createTestUser();
+			const userId = user._id;
+			await testObjects.createTestProblem({ userId });
+			const result = await problemRepo.deleteProblemsForUser(userId);
+			expect(result.success).to.be.equal(true);
+			expect(result.deletedDocuments).to.be.equal(1);
+		});
+
+		it('when problem is deleted, it should be gone from db', async () => {
+			const user = await testObjects.createTestUser();
+			const userId = user._id;
+			const problemUser1 = await testObjects.createTestProblem({ userId });
+
+			const user2 = await testObjects.createTestUser();
+			const userId2 = user2._id;
+			const problemUser2 = await testObjects.createTestProblem({ userId: userId2 });
+
+			let userProblems = await problemRepo.getProblemsForUser(userId);
+			expect(userProblems[0]._id.toString()).to.be.equal(problemUser1._id.toString());
```

Comment:
```suggestion
			expect(userProblems).to.be.an('array').of.length(1);
			expect(userProblems[0]._id.toString()).to.be.equal(problemUser1._id.toString());
```
just check length 1 here and after L55 too